src/main/java/net/rakugakibox/springbootext/logback/access/tomcat/RequestAttributeRegardingAccessEvent.java

```diff
@@ -0,0 +1,91 @@
+package net.rakugakibox.springbootext.logback.access.tomcat;
+
+import ch.qos.logback.access.spi.AccessEvent;
+import ch.qos.logback.access.spi.ServerAdapter;
+import org.apache.catalina.AccessLog;
+
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+/**
+ * AccessEvent that regards requestAttributes that may be set by other valves.
+ */
+public class RequestAttributeRegardingAccessEvent extends AccessEvent {
+
+    private String remoteAddress;
+    private String remoteHost;
+    private String protocol;
+    private Integer localPort;
+
+    public RequestAttributeRegardingAccessEvent(HttpServletRequest httpRequest, HttpServletResponse httpResponse,
+                                                ServerAdapter adapter) {
+        super(httpRequest, httpResponse, adapter);
+        prepareForDeferredProcessing();
```

Comment:
Can I ask you why call `prepareForDeferredProcessing()`?
I think it is better to be lazy, like the parent class.
I passed the test with it.